app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,85 @@
+class GroupsController < ApplicationController
+  def index
+    # currently also using this endpoint to pull down suggested groups
+    groups = if params[:limit].present? # Get a specific amount of trending
+      Group.trending(params[:limit].to_i)
+    elsif params[:trending].present? # Get all trending groups, paginated
+      Group.trending(0).page(params[:page]).per(20)
```

Comment:
Also I don't believe this will actually work.

`Group.trending(0)` will do a Redis lookup that returns the IDs of _all_ the trending groups. Then we do a `Group.where(id: <ids>)` which can return results in any order.